packages/flutter_migrate/lib/src/compute.dart

```diff
@@ -55,11 +55,8 @@ bool _skipped(String localPath, FileSystem fileSystem,
     }
   }
   if (skippedPrefixes != null) {
-    final Iterable<String> canonicalizedSkippedPrefixes =
```

Comment:
Don't canonicalize, as it attempts to convert to absolute paths and this interferes with prefixing.

Reply:
would we be susceptible to differences in path separator or ASCII case?

Reply:
Let me try normalize(), maybe that's what we are looking for.

Reply:
I think a combination of path separator conversion and the normalize() function should be safe